package.json

```diff
@@ -94,7 +94,7 @@
     "autoprefixer-loader": "2.0.0",
     "babel-cli": "6.18.0",
     "babel-core": "6.21.0",
-    "babel-jest": "20.0.0",
+    "babel-jest": "^20.0.3",
```

Comment:
We're locking the rest of our dependencies to specific versions. Any reason to deviate for these dependencies?